docs/Style-Guide.md

```diff
@@ -0,0 +1,157 @@
+
+<br>
+
+### Get straight to the point
+
+<br>
+
+Give your readers a clear and precise action to take.
+
+<br>
+
+### Avoid adding video or image content 
+
+<br>
+
+Do not add videos or screenshots. Instead, make a referral link or a youtube video.
+
+<br>
+
+### Avoid plagarism 
+
+<br>
+
+Make sure you evade copying other people's work, keep it as original as possible. You can learn from what they've done and also reference where it is if you used a particular quote from their work.
+
+<br>
+
+<br>
+
+## Word Choice
+
+<br>
+
+<!-- content here -->
+
+There are a few things you need to use and avoid when writing your documentation which will improve readability to your reader and also make your documentation neat, direct and clean all through your writing process.
+
+
+<br>
+
+### Use "you" instead of we or they
+
+
+<br>
+
+<!-- content here -->
+
+Using the word "you" makes the reader know you are speaking to them. It is easier to give them direct instruction on what to do when a particular topic. 
+
+ > According to [Wikipedia](#), ***You*** is usually a second person pronoun. Also, used to refer to an indeterminate person, as a more common alternative to a very formal indefinite pronoun.
+
+<br>
+
+
+### Avoid words like "*just*" or  "*simply"
+
+
+<br>
+
+<!-- content here -->
+
+The reader may not find it easy to use the Fastify's framework and plugins, so avoid words that make it sound like they are simple or easy. Not everyone who reads the documentation has the same level of understanding.
+
+<br>
+
+### Starting with a verb
+
+<br>
+<!-- content here -->
+
+Mostly start your description with a verb, which makes it simple and precise for the reader to follow. Use present tense in more often as it is easier to read and understand than the past or future tense.
+
+Example
+
+- [ ] Less like this: There is a need for Nodejs to be installed before you can be able to use fastify.
+
+- [x] More like this: Install Nodejs to make use of fastify 
+
+<br>
+
+### Grammatical moods 
+
+<br>
+
+<!-- content here -->
+
+Grammatical moods is a great way to express your writing, don't want to sound too bossy while making a direct statement. Know when to switch between indicative, imperative, and Subjunctive moods.
+
+<br>
+
+**Indicative** - Use when making a factual statement or question. 
+
+Example: Since there is no testing framework available, "we do recommend ways to write your test".
+
+<br>
+
+**Imperative** - Use when giving instructions, actions, commands, or when you write your headings.
+
+Example: Install dependencies before you start application development mode.
+
+<br>
+
+**Subjunctive** -  Use when making suggestions, hypothesis or non-factual statement.
+
+Example: We recommend you read the documentation on our website to get full information on the plugins.
```

Comment:
All those examples use "we" instead of "you"

Reply:
Thanks for pointing that out, I'll rephrase this example.